package.json

```diff
@@ -86,8 +86,8 @@
     "webpack-dev-server": "^3.1.3"
   },
   "resolutions": {
-    "lodash": "4.17.21",
-    "hosted-git-info": "4.0.2",
-    "url-parse": "1.5.1"
+    "lodash": "^4.17.21",
+    "hosted-git-info": "^4.0.2",
+    "url-parse": "^1.5.1"
   }
```

Comment:
This should do the trick?

Reply:
blast the node_modules then try it out locally :)

Reply:
Blasted and tested✅ 
Works as expected